rpcs3/Emu/RSX/Program/GLSLCommon.cpp

```diff
@@ -209,220 +119,22 @@ namespace glsl
 			{ "VTX_FMT_SNORM16", RSX_VERTEX_BASE_TYPE_SNORM16 },
 			{ "VTX_FMT_FLOAT32", RSX_VERTEX_BASE_TYPE_FLOAT },
 			{ "VTX_FMT_FLOAT16", RSX_VERTEX_BASE_TYPE_HALF_FLOAT },
-			{ "VTX_FMT_UNORM8", RSX_VERTEX_BASE_TYPE_UNORM8 },
-			{ "VTX_FMT_SINT16", RSX_VERTEX_BASE_TYPE_SINT16 },
-			{ "VTX_FMT_COMP32", RSX_VERTEX_BASE_TYPE_CMP32 },
-			{ "VTX_FMT_UINT8", RSX_VERTEX_BASE_TYPE_UINT8 }
+			{ "VTX_FMT_UNORM8 ", RSX_VERTEX_BASE_TYPE_UNORM8 },
+			{ "VTX_FMT_SINT16 ", RSX_VERTEX_BASE_TYPE_SINT16 },
+			{ "VTX_FMT_COMP32 ", RSX_VERTEX_BASE_TYPE_CMP32 },
+			{ "VTX_FMT_UINT8  ", RSX_VERTEX_BASE_TYPE_UINT8 }
```

Comment:
why spaces in the strings?

Reply:
To make the generated code easier to read.